openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx

```diff
@@ -381,7 +379,9 @@ const TestSuitePipelineTab = () => {
         dataIndex: 'pipelineStatuses',
         key: 'recentRuns',
         render: (_, record) => (
-          <Row align="middle">{getIngestionStatuses(record)}</Row>
+          <Row align="middle">
+            <IngestionRecentRuns ingestion={record} />
+          </Row>
```

Comment:
The row should have col as direct children.

Reply:
Will do it next